msal/src/test/java/com/microsoft/identity/client/e2e/tests/network/nativeauth/SignUpTest.kt

```diff
@@ -48,6 +48,29 @@ class SignUpTest : NativeAuthPublicClientApplicationAbstractTest() {
         Dispatchers.setMain(testDispatcher)
     }
 
+    private fun <T> retryOperation(
```

Comment:
Let's move this into a separate file, as we're going to need to re-use this in many tests.

Reply:
Removed it to NativeAuthPublicClientApplicationAbstractTest as Robert suggested.